src/coredump/coredumpctl.c

```diff
@@ -324,6 +324,7 @@ static int print_list(FILE* file, sd_journal *j, int had_legend) {
                 retrieve(d, l, "COREDUMP_COMM", &comm);
                 retrieve(d, l, "COREDUMP_CMDLINE", &cmdline);
                 retrieve(d, l, "COREDUMP_FILENAME", &filename);
+                retrieve(d, l, "COREDUMP", &coredump);
```

Comment:
Since I don't know how this is related to the `sd_j_set_data_threshold()` threshold you mention in the commit message: This makes sure to only get the first few kB of the core dump, not the full one? Or is that not the actual core dump but just a flag whether it's in the journal?